### DISCUSSION TITLE
Super Tux Kart

### DISCUSSION THREAD

**Discussion by Tskk-dev** [skyline-emu/skyline#907] [Game Requests]
because its super tux cart. 
![image](https://user-images.githubusercontent.com/96981837/161698917-29bc10f4-f073-4c2b-aa23-b6affd16ce17.png)

**Comment by Swizzlefowl** [comment]
This game made me racist 10/10.

**Comment by riperiperi** [comment]
https://user-images.githubusercontent.com/6294155/161753368-7b5b0706-2649-417d-beb9-068281fe7e6a.mp4

**Reply by Tskk-dev** [reply]
lmfao

**Comment by PixelyIon** [comment]
Super Tux Kart is truly the magnum opus of gaming, a cultural landmark.. Skyline strives to one day support the advanced rendering pipeline inside Super Tux Kart to render graphics more beautiful than any artist could even imagine. We all patiently wait for the day when you can Tux on the Go (TotG) using Skyline.
